Source/ZoomNet/Resources/Groups.cs

```diff
@@ -39,6 +40,19 @@ public async Task<bool> AddUserToGroupAsync(string groupId, string userEmail, Ca
 			.AsMessage()
 			.ConfigureAwait(false);
 
-		return response.IsSuccessStatusCode;
+		if (!response.IsSuccessStatusCode)
```

Comment:
No need to check `IsSuccessStatusCode` because the library is already designed to throw an exception when a "non-success" status code is detected.